internal/kms/keyprotect.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	kmsTypeKeyProtectMetadata = "kp-metadata"
+	kmsTypeKeyProtectMetadata = "ibmkeyprotect"
```

Comment:
This breaks deployments in 3.5 that use this.

Reply:
sure, added backporting label. @nixpanic